src/main/kotlin/org/jetbrains/research/libsl/nodes/references/TypeReference.kt

```diff
@@ -3,11 +3,11 @@ package org.jetbrains.research.libsl.nodes.references
 import org.jetbrains.research.libsl.context.LslContextBase
 import org.jetbrains.research.libsl.type.*
 
-open class TypeReference(
```

Comment:
Shouldn't we also add something like `WildcardTypeReference` for "`?`" explicit use-cases?
(instead of hard-coded special checks on `name` field)

Reply:
Was introduced new TypeRefenece: WildcardTypeReference.

Fixed;